tests/unit/test_runner.py

```diff
@@ -168,8 +168,8 @@ def test_run_app_for_input_output_path_with_space(
     expected_container_output /= app_manifest["working-directory"]
 
     returncode = runner.run_app(sample_map_name, input_path, output_path, app_manifest, faux_pkg_manifest, quiet)
-    input_path_with_quotes = f'"{input_path.absolute()}"'
-    output_path_with_quotes = f'"{output_path.absolute()}"'
+    input_path_with_quotes = '"' + f"{input_path.absolute()}" + '"'  # To quiet flake8 complaint
+    output_path_with_quotes = '"' + f"{output_path.absolute()}" + '"'
```

Comment:
Looks good (I am okay as it is).

Just wonder if

```
    input_path_with_quotes = f'''"{input_path.absolute()}"'''
    output_path_with_quotes = f'''"{output_path.absolute()}"'''
```
shows the same flake8 complaints.

```
    input_path_with_quotes = '"' + str(input_path.absolute()) + '"'  # To quiet flake8 complaint
    output_path_with_quotes = '"' + str(output_path.absolute()) + '"'
```
Above also can do the same.

Reply:
Yes, yet another way that works, in addition to str.format(), a leading or trailing space, but escape does not work as black would complain and remove it. Thx.